x/clp/abci.go

```diff
@@ -136,3 +137,17 @@ func BeginBlocker(ctx sdk.Context, k kpr.Keeper) {
 		panic(err)
 	}
 }
+
+var blockTime *time.Time = nil
+
+func MeasureBlockTime(ctx sdk.Context, k kpr.Keeper) {
+	now := time.Now()
+	if blockTime == nil {
+		blockTime = &now
+		return
+	}
+
+	elapsed := now.Sub(*blockTime)
+	blockTime = &now
+	k.Logger(ctx).Info(fmt.Sprint("Block took ", elapsed.Seconds(), "s to execute"))
```

Comment:
Might want to put the block number in this log too.

Reply:
Block height is printed in the very next line. I think it's fine to not include it here
```
12:29PM INF Block took 5.014953372s to execute module=x/clp
12:29PM INF executed block height=4 module=state num_invalid_txs=0 num_valid_txs=0
```